src/components/views/rooms/RoomTile2.tsx

```diff
@@ -233,13 +232,22 @@ export default class RoomTile2 extends React.Component<IProps, IState> {
             'mx_RoomTile2_minimized': this.props.isMinimized,
         });
 
-        const badge = (
-            <NotificationBadge
+        const avatarSize = 32;
+        const roomAvatar = <DecoratedRoomAvatar
```

Comment:
`avatarSize` and `roomAvatar` are only used once.
I wonder if it's worth declaring variables for them instead of directly putting their values where they are being used.

Reply:
Good idea

Reply:
I'd like to keep roomAvatar where it is because it's easier to follow the logic for where the bade notification is displayed